pydantic_extra_types/types/money.py

```diff
@@ -0,0 +1,1468 @@
+"""all supported currencies, Data about currencies is taken from https://github.com/sebastianbergmann/money"""
```

Comment:
this repo is archived.

If we're to support this type, we need:
1. a reliable up-to-date source
2. a way to update our list with a script from the source.